server/helpers/sendEmail.js

```diff
@@ -14,13 +14,15 @@ sgMail.setApiKey(process.env.SENDGRID_API_KEY);
  * @returns {object} object
  */
 
-  const sendEmail = (to, token, hostUrl, template) => {
-    const { subject, from, text, html } = template;
-    const url =
+  const sendEmail = (to, token, hostUrl, template, redirectUrl) => {
+    const { subject, from, text } = template;
+    let { html } = template;
```

Comment:
Why is the action on line 19 happening separately from line 18?

Reply:
The action is separated because it is not a const. It was modified